src/main/java/core/basesyntax/Circle.java

```
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea, Draw {
+    private double radius;
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {

```

The setRadius method should throw an exception like `IllegalArgumentException` if the radius is not greater than 0. This will prevent the creation of invalid Circle objects.